# Generated by Django 4.2.8 on 2024-01-06 17:10

from django.conf import settings
from django.db import migrations, models
import django.db.models.deletion
import django.utils.timezone


class Migration(migrations.Migration):

    dependencies = [
        migrations.swappable_dependency(settings.AUTH_USER_MODEL),
        ('techdocs', '0004_productionline_valid_record_exported'),
    ]

    operations = [
        migrations.AlterModelOptions(
            name='components',
            options={'ordering': ['componentName'], 'verbose_name_plural': 'Components'},
        ),
        migrations.AlterModelOptions(
            name='machineissues',
            options={'ordering': ['issueName'], 'verbose_name_plural': 'Machine issues'},
        ),
        migrations.AlterModelOptions(
            name='machineworks',
            options={'ordering': ['workListItemName'], 'verbose_name_plural': 'Machine work'},
        ),
        migrations.AlterModelOptions(
            name='productionline',
            options={'ordering': ['lineName']},
        ),
        migrations.RenameField(
            model_name='record',
            old_name='exported',
            new_name='confirmed',
        ),
        migrations.AddField(
            model_name='record',
            name='confirmed_at',
            field=models.DateTimeField(default=django.utils.timezone.now),
        ),
        migrations.AddField(
            model_name='record',
            name='confirmed_by',
            field=models.CharField(max_length=255, null=True),
        ),
        migrations.AlterField(
            model_name='record',
            name='user',
            field=models.ForeignKey(null=True, on_delete=django.db.models.deletion.DO_NOTHING, to=settings.AUTH_USER_MODEL),
        ),
    ]
